compaction/time_window_compaction_strategy.cc

```diff
@@ -233,6 +233,7 @@ time_window_compaction_strategy::get_reshaping_job(std::vector<shared_sstable> i
 
     size_t offstrategy_threshold = std::max(schema->min_compaction_threshold(), 4);
     size_t max_sstables = std::max(schema->max_compaction_threshold(), int(offstrategy_threshold));
+    const uint64_t target_job_size = cfg.free_storage_space * reshape_target_space_overhead;
```

Comment:
Should we take sharding into account?
Or this 10% is already conservative enough that multiple shards reshaping in parallel will not be a problem/

Reply:
Oops I forgot to do it. I think we should